build/repo.targets

```diff
@@ -69,7 +69,7 @@
 
     <ItemGroup>
       <PackageStoreManifestFiles Include="$(PackageCacheOutputPath)**\artifact.xml">
-        <DestinationFile>manifest.$(RID).xml</DestinationFile>
+        <DestinationFile>aspnetcore.manifest.$(RID).xml</DestinationFile>
```

Comment:
This needs to match what I put in the issue, i.e. `aspnetcore-store-$(VERSION)-$(RID).xml`